templates/harbor-kubedb.yaml

```diff
@@ -1,5 +1,10 @@
 {{- if and .Values.harbor.create .Values.kubedb.create }}
 
+{{- $backupsEnabled := false }}
+{{- with .Values.harbor.dbBackups }}
+  {{ $backupsEnabled = and .bucketName .credentials.accessKeyId .credentials.secretAccessKey }}
+{{- end }}
```

Comment:
I would give them a `habor.dbBackups.enabled` defaulted to `true` and then just the let the chart blow up on them if there aren't any credentials.

Reply:
@jwietelmann Is there an expectation to make the defaults installable? My intuition is to make `enabled: true` the default b/c that's the "right" thing to do, but then you need to fill in the other stuff.

Reply:
I didn't read your comment carefully enough. It must be OK to default to `true` b/c that's what you said to do

Reply:
Yeah, just to explain my thought process: I want installation to fail if they don't provide config for backups, and barring that I want them to have to explicitly say "No, don't back up my database." MDE should make it as easy as possible to do the right thing and make it easy but _explicit_ to do the not-so-right thing.